incubator/spring-cloud-data-flow/templates/server-rbac.yaml

```diff
@@ -2,8 +2,13 @@
 kind: Role
 apiVersion: rbac.authorization.k8s.io/{{ required "A valid .Values.rbac.apiVersion entry required!" .Values.rbac.apiVersion }}
 metadata:
-  name: {{ template "fullname" . }}-role
+  name: {{ template "fullname" . }}
```

Comment:
I wonder why this resource explicitly specifies the namespace as opposed to the other resources in the chart. I'd suggest you remove it.